src/main/java/ledger/user_interface/ui_controllers/AccountPopupController.java

```diff
@@ -52,18 +52,19 @@
     @Override
     public void initialize(URL fxmlFileLocation, ResourceBundle resources) {
         this.submitAccountInfo.setOnAction((event) -> {
-            try {
-                Account account = getAccountSubmission();
-                AccountBalance balance = getAccountBalance();
-
+            Account account = getAccountSubmission();
+            AccountBalance balance = getAccountBalance();
+            if (account != null && balance != null) {
                 TaskWithArgs<Account> task = DbController.INSTANCE.insertAccount(account);
                 task.RegisterSuccessEvent(this::insertDone);
                 task.RegisterFailureEvent(this::insertFail);
                 task.startTask();
-            } catch (NullPointerException e2) {
-                this.setupErrorPopup("Required field is null.", e2);
-            } catch (NumberFormatException e3) {
-                this.setupErrorPopup("Account starting amount must be an integer.", e3);
+                return;
+            }
+            if (account==null) {
```

Comment:
Here's another one.  I think there's a shortcut for formatting entire files at once that will take care of things like these.